### PR TITLE
Initial Upload

### PR DESCRIPTION
**Pull request by KyleAure**
Initial draft of sessions guide for review.

### CODE REVIEW

File: finish/src/main/liberty/config/server.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <feature>jaxrs-2.0</feature>
+        <feature>mpOpenAPI-1.0</feature>
+    </featureManager>
+
+    <httpEndpoint httpPort="${default.http.port}" httpsPort="${default.https.port}"
+        id="defaultHttpEndpoint" host="*" />
+
+    <!-- tag::jcachelib[] -->
+    <library id="jCacheVendorLib">
+	     <fileset dir="${wlp.install.dir}../../../src/main/resources/" includes="hazelcast-3.9.2.jar"/>
```
[the remark this answers is not in the document]
Comment (KyleAure) [reply finish/src/main/liberty/config/server.xml]: Hey Andy,
You will probably need to help me with this.  I haven't done much with bootstrap properties.
Thanks!

**Comment by evelinec** [comment]
@JunyiSun can you review this PR? Thank you.

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+then that server's session data will be lost.
+
+image::sessionCache.png[]
+
+`Session persistence` using a `session cache`, as illustrated above, solves this
+problem by allowing all server instances to share caches amongst each other. This
+eliminates the need to always route a user to the same server instance, and also
+helps in failover situations by distributing the cache.
+
+== Building and running the application
```
Comment (KyleAure) [inline README.adoc]: @JunyiSun The `Building and running the application` section starts on line 95.

Comment (JunyiSun) [reply README.adoc]: @KyleAure Hey I realized that your `Building and running the application` does not mention anything about how to compile and run the application, which does not serve the purpose of this section. You can take a look at how our common  `Building and running the application` section does here: https://raw.githubusercontent.com/OpenLiberty/guides-common/master/mvnbuild.adoc

Comment (JunyiSun) [reply README.adoc]: We use shared common file in our guides. As an example, in the https://github.com/OpenLiberty/guide-cdi-intro, you can see how mvnbuild.adoc, mvnverify.adoc files get linked there. We should follow this style to make our guides consistent. Thank you.

**Comment by KyleAure** [comment]
@JunyiSun Sorry for the misunderstanding earlier.  I edited the previous section to be `Creating the application` and then added the `building and running the application` section before testing.  I wasn't able to use the common section because in this case the user needs to start two servers instead of just one.

**Pull request merged**